Changelog.md

```diff
@@ -4,6 +4,10 @@
 
 - Completion for `open-api` option in Camel K modeline filtered with JSON and YAML files
 - Completion for resource-like options (`open-api`, `property-file` and `resource`) in Camel K modeline is hiding IDE configuration files (which are in `.vscode` folder)
+- Document symbols for Camel routes written with Java DSL. It implies these features:
+  - Outline. Nota: if you select the sort by position, it allows to have an overview of the route flow, not only of the hierarchy of elements.

```

Comment:
Nota? I would just say "Outline. Note that if you select 'sort by position', you gain an overview of the route flow, not only of the element hierarchy."